program/Project Descriptions and Plans/SDK/sdk-project-concept.md

```diff
@@ -133,7 +141,7 @@ Milestones:
 
 1. OpenHW IDE and debugger:
 
-   - CORE-V MCU dev board integrated for Eclipse CDT debugging.
+   - CORE-V MCU development board integrated for Eclipse CDT debugging.
```

Comment:
We need to list the boards that is planned to be supported for "version 1"

Reply:
I think this is clear, there is the one true board. Is there a better wording I should use?

Reply:
Yes, I think the exact name of the board whould be helpful.